src/plugins/saved_objects_management/public/management_section/recent_work.tsx

```diff
@@ -208,47 +230,81 @@ export const RecentWork = (props: { core: CoreStart; workspaceEnabled?: boolean
                 core.http.basePath,
                 core.application.navigateToUrl
               );
+
               content = (
                 <EuiCard
-                  title={recentAccessItem.label}
-                  titleSize="xs"
+                  title={
+                    <EuiFlexGroup justifyContent="flexStart" alignItems="center" gutterSize="none">
+                      <EuiFlexItem grow={false}>
+                        <EuiIcon
+                          style={{ marginRight: widthForRightMargin }}
+                          type={recentAccessItem.meta.icon || 'apps'}
+                        />
+                      </EuiFlexItem>
+                      <EuiFlexItem grow={false}>
+                        <EuiText size="xs" color="subdued">
+                          {recentAccessItem.type.charAt(0).toUpperCase() +
+                            recentAccessItem.type.slice(1)}
```

Comment:
Could you elaborate more on it that why we do this? I suppose this wants to make the first letter of type to uppercase to display, right? Could we add an annotation for this or extract a new function to do this?

Reply:
Sure, I will extract a new function for it. Thank you